src/tests/utils/stateUtils.tsx

```diff
@@ -0,0 +1,25 @@
+import { ModalActionMenuVisibilityState } from "../../types/reducers/ModalActionMenuVisibilitySlice";
+
+export function generateModalActionsVisibilityState(): ModalActionMenuVisibilityState {
```

Comment:
As part of the tests for components under CardModalActions, you need to specify he ModalActionMenuVisibility state object. In each test, we only check that one menu is open, and it's pretty tedious and confusing to have to define the whole state in the test. Therefore, I created this helper to generate the default state, and any modifications to this state can happen in the test, reducing the amount of code we write and the amount of unnecessary code in the tests